File: Source/Fantasy.Jobs/Scheduling/InlineAction.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Fantasy.XSerialization;
using System.Xml.Linq;

namespace Fantasy.Jobs.Scheduling
{
    [DataContract]
    [XSerializable("inlineAction")]
    public class InlineAction : Action, IXSerializable 
    {

        public override ActionType Type
        {
            get { return ActionType.Inline; }
        }

        [DataMember]
        public string Xslt { get; set; }

        #region IXSerializable Members

        public void Load(IServiceProvider context, XElement element)
        {
            XHelper.Default.LoadByXAttributes(context, element, this);
            XElement xsltElement = element.Elements().FirstOrDefault();
            this.Xslt = xsltElement != null ? xsltElement.ToString() : null;
        }

        public void Save(IServiceProvider context, XElement element)
        {
            XHelper.Default.SaveByXAttributes(context, element, this);
            if (!string.IsNullOrEmpty(Xslt))
            {
                element.Add(XElement.Parse(Xslt));
            }
        }

        #endregion
    }
}

File: AutoConverted/Fantasy.Jobs/Scheduling/InlineAction.java
package Fantasy.Jobs.Scheduling;

import Fantasy.XSerialization.*;

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
//[DataContract, XSerializable("inlineAction")]
public class InlineAction extends Action implements IXSerializable
{

	@Override
	public ActionType getType()
	{
		return ActionType.Inline;
	}

//C# TO JAVA CONVERTER TODO TASK: Java annotations will not correspond to .NET attributes:
	//[DataMember]
	private String privateXslt;
	public final String getXslt()
	{
		return privateXslt;
	}
	public final void setXslt(String value)
	{
		privateXslt = value;
	}

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#region IXSerializable Members

	public final void Load(IServiceProvider context, XElement element)
	{
		XHelper.Default.LoadByXAttributes(context, element, this);
		XElement xsltElement = element.Elements().FirstOrDefault();
		this.setXslt(xsltElement != null ? xsltElement.toString() : null);
	}

	public final void Save(IServiceProvider context, XElement element)
	{
		XHelper.Default.SaveByXAttributes(context, element, this);
		if (!DotNetToJavaStringHelper.isNullOrEmpty(getXslt()))
		{
			element.Add(XElement.Parse(getXslt()));
		}
	}

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#endregion
}